```
/home/fabian/projects/zephyr-workspace/zephyr/modules/lvgl/lvgl_zephyr_osal.c:20:73: error: unknown type name ‘lv_thread_prio_t’; did you mean ‘lv_thread_sync_t’?
   20 | lv_result_t lv_thread_init(lv_thread_t *thread, const char *const name, lv_thread_prio_t prio,
      |                                                                         ^~~~~~~~~~~~~~~~
      |                                                                         lv_thread_sync_t
/home/fabian/projects/zephyr-workspace/zephyr/modules/lvgl/lvgl_zephyr_osal.c:160:6: error: ‘thread_entry’ defined but not used [-Werror=unused-function]
  160 | void thread_entry(void *thread, void *cb, void *user_data)
      |      ^~~~~~~~~~~~
cc1: all warnings being treated as errors
```
Tests enabling OSAL fail. The symbol still exists:
https://github.com/lvgl/lvgl/blob/dc9e9dee5eebe7251dbe36b5e1731ad7a5166fe5/src/osal/lv_os_private.h#L59

Seems to be an issue around the private header inclusion.

--- a/west.yml
+++ b/west.yml
@@ -321,7 +321,7 @@ manifest:
       revision: fb00b383072518c918e2258b0916c996f2d4eebe
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: 85aa60d18b3d5e5588d7b247abf90198f07c8a63
+      revision: bbedf2656086a93a8468a1c98168a6b36631ce9a
       path: modules/lib/gui/lvgl
     - name: mbedtls
       revision: 554163f6f7b2ec71fed9b423e81f2217a8615d29

